public/config.js

```diff
@@ -0,0 +1,236 @@
+ CONFIG = {
+     basename: '/',
+     apps: [{
+             path: '/',
+             display: 'Flux Notes™',
+             app: "LandingPage",
+             isExact: true
+         },
+         {
+             path: '/launch',
+             display: 'Flux',
+             app: "LaunchPage",
+             isExact: true,
+             launchContext: {
+                 client: {
+                     client_id: '6c12dff4-24e7-4475-a742-b08972c4ea27',
```

Comment:
Is there any potential concern from having the `client_id` be public? By my reading of the SMART App Launch Framework it sounds like it should be ok, just want to double check.

Reply:
There should be no problem with this, authentication is actually at the user level in this regard and the id just identifies that application being used.  The id coupled with the redirection url must match what the server has for the client as well.